### DISCUSSION TITLE
Direct link to Presets/Assistants?

### DISCUSSION THREAD

**Discussion by stanpete78** [danny-avila/LibreChat#3635] [Q&A] [answered]
Hi!
I've set up LibreChat for my family and it's working great. To make things easier for my parents it would be great to have direct links to some of the assistants instead of having to navigate through the menus.
For example something like "baseurl.com/c/assistant1" gets me to a specific preset/assistant, so they can save them as bookmarks in their browser.

So my question is: Is this possible somehow?

**Comment by airwindk** [comment]
This would be incredibly useful especially since it allows the assistant to be directly embedded as a widget in another website. Wondering if there have been any updates on this?

**Comment by leeks8888** [comment, marked as answer]
​Yes, LibreChat allows you to create direct URLs that open a new chat with a specific agent. 
This is accomplished by using the agent_id query parameter in the URL.​
[GitHub](https://github.com/danny-avila/LibreChat/discussions/6361?utm_source=chatgpt.com)

🔗 Example
To initiate a new chat with a specific agent, use the following URL structure:​

https://your-librechat-domain.com/c/new?agent_id=your-agent-id
Replace your-librechat-domain.com with your actual LibreChat domain.

Replace your-agent-id with the unique identifier of the agent you wish to use.​

This URL will start a new conversation with the specified agent. The endpoint is automatically set to agents when using this parameter.

**Comment by stanpete78** [comment]
@leeks8888 Thanks!
After your post I realised that you now have more options to choose from. 
I use Nginx Proxy Manager and I managed to always open my preferred model when using the base url gpt.myurl.com with this simple entry in the Custom Nginx Configuration field:

`if ($request_uri = "/") {
    return 302 /c/new?endpoint=openAI&model=gpt-4.1-mini;
}`

So now I can always set a default "best" model for my family. In order for this to work, you need to unpin your own default preset in LibreChat. If you don't, your default preset will be chosen instead.
